### ISSUE TITLE
syntax highlighting on ocaml breaks with %s in string

### ISSUE THREAD

**Issue by tcoopman** [onivim/oni2#1692]
```ocaml
let () = "test %s" x
  foo
```

everything after the %s is seen is highlighted as a string as you can see here:
![2020-04-30-151127_233x62](https://user-images.githubusercontent.com/45546/80714280-009aff80-8af5-11ea-854d-08aab0005e85.png)

**Comment by glennsl** [comment]
Weird. I can't reproduce this with reason-vscode directly in vscode, and updating the ocaml syntax to the latest from the rls repository didn't fix it. So looks like this might be an issue with the syntax highlighter itself.

**Comment by bryphe** [comment]
Indeed, looks like a bug in [`reason-textmate`](https://github.com/onivim/reason-textmate) - we should add an OCaml test case and see what's going on.

I tried out an alternate syntax here: https://github.com/hackwaly/vscode-ocaml/tree/master/syntaxes
(They are `tmLanguage`/xml which we previously could not load - but @glennsl recently added this in https://github.com/onivim/oni2/pull/1616)

and it _seems_ to work better for me, at least in this case:
![image](https://user-images.githubusercontent.com/13532591/80726167-e6dbc700-8ab8-11ea-8e26-4fd851b1c79e.png)

larger files seem to look OK too:
![image](https://user-images.githubusercontent.com/13532591/80726239-02df6880-8ab9-11ea-9c74-cf62af7540c3.png)

So perhaps in the short-term - we can port over the ocaml syntaxes from that repo to use, until we get a chance to fix the `reason-textmate` bug.
